Src/PChecker/CheckerCore/CheckerCore.csproj

```diff
@@ -22,4 +22,9 @@
         <None Include="../../../Icon/icon.png" Pack="true" PackagePath="" />
         <None Include="../../../README.md" Pack="true" PackagePath="" />
     </ItemGroup>
+    <ItemGroup>
+        <Reference Include="PCSharpRuntime">
+            <HintPath>..\..\..\Tutorial\1_ClientServer\PGenerated\CSharp\net6.0\PCSharpRuntime.dll</HintPath>
```

Comment:
First, we shouldn't include stuff from PGenerated/ folder.

Second, importing PCSharpRuntime in CheckerCore can cause cyclic dependency since PCSharpRuntime imports CheckerCore: https://github.com/p-org/P/blob/master/Src/PRuntimes/PCSharpRuntime/CSharpRuntime.csproj#L25